### DISCUSSION TITLE
QR code de equipamentos

### DISCUSSION THREAD

**Discussion by minuanorefrigera** [RamonSilva20/mapos#2021] [Ideas]
Gostaria de deixar uma sugestão
Gerar um QR code que possa ser impresso e colado nos aparelhos do cliente

**Comment by elsmmm** [comment]
seria muito pratico onde iria no qr cod a tag do equipamrnto

**Comment by willph** [comment]
> Gostaria de deixar uma sugestão Gerar um QR code que possa ser impresso e colado nos aparelhos do cliente

O intuito seria identificar de forma mais fácil a quem ele pertence, com Nº OS e nome do cliente?

**Comment by minuanorefrigera** [comment]
Isso, ou, em um local com vários aparelhos saber tudo o que já foi feito
Exemplo: cliente X tem 20 aparelhos de ar condicionado
Poder gerar os QR code de cada aparelho para um único cliente com as
informações do que já foi feito no aparelho

Em seg, 15 de mai de 2023 15:56, Wilmerson ***@***.***>
escreveu:
